src/IssueViz.UnitTests/Models/AnalysisIssueLocationVisualizationTests.cs

```diff
@@ -51,11 +51,65 @@ public void Ctor_InitialFilePathIsTakenFromIssueLocation()
         [TestMethod]
         public void Ctor_InitialSpanIsNull()
         {
-            var testSubject = CreateTestSubject();
+            var testSubject = CreateTestSubject(filePath:"test");
```

Comment:
It was previously null, which according to the logic, will now set the span to Empty. This way the test checks the previous behavior, of initial state when the file name is not null.